services-camel/src/test/java/edu/unc/lib/dl/services/camel/routing/MetaServicesRouterTest.java

```diff
@@ -77,8 +80,7 @@ public void testRouteStartContainer() throws Exception {
 
     @Test
     public void testEventTypeFilter() throws Exception {
-        getMockEndpoint("mock:{{cdr.enhancement.stream.camel}}").expectedMessageCount(0);
```

Comment:
Should there still be a test for this endpoint somewhere?

Reply:
The route being tested here doesn't go to that destination anymore. There is a test that checks for messages going to `{{cdr.enhancement.stream.camel}}` down in the test of the `process.creation` route though:
https://github.com/UNC-Libraries/Carolina-Digital-Repository/pull/1002/files/810b15210ea2ce06832543eff1aa46f34fe52367#diff-490c86aa5ca6fd996958ce730fc402a0R132